cli/src/cli_util.rs

```diff
@@ -1535,6 +1535,22 @@ to the current parents may contain changes from multiple commits.
         self.commit_summary_template().format(commit, formatter)
     }
 
+    pub fn write_stale_commit_stats(
```

Comment:
nit: perhaps, we can extract `print_update_stats()` or something from `update_working_copy()`. It's not important that this function doesn't print the wc parents.

(can be addressed later)

Reply:
Left a TODO for now.